src/applications/static-pages/widgetTypes.js

```diff
@@ -1,8 +1,8 @@
 export default {
   BURIALS_APP_STATUS: 'burials-app-status',
```

Comment:
These are separate widget types than the react widgets this PR is modifying. I just alphabetized this list here 🙂